code/game/objects/items/devices/flash.dm

```diff
@@ -93,6 +93,7 @@
 						H.eye_blurry = max(H.eye_blurry, flash_strength + 5)
 						H.flash_eyes()
 						H.adjustHalLoss(halloss_per_flash * (flash_strength / 5)) // Should take four flashes to stun.
+						H.apply_damage(flash_strength * H.species.flash_burn/5, BURN, BP_HEAD, 0, 0, "Highly abnormal growths")

```

Comment:
Might want to make this a more generic burn message.

Reply:
...oh right I made this generic didn't I